cli/src/helpers/logNextSteps.ts

```diff
@@ -23,7 +23,9 @@ export const logNextSteps = async ({
   const pkgManager = getUserPkgManager();
 
   logger.info("Next steps:");
-  projectName !== "." && logger.info(`  cd ${projectName}`);
+  if (projectName !== ".") {
+    logger.info(`  cd ${projectName}`);
+  }
```

Comment:
[`@typescript-eslint/no-unused-expressions`](https://typescript-eslint.io/rules/no-unused-expressions) was promoted to the [`recommended` preset](https://v8--typescript-eslint.netlify.app/users/configs#recommended) in v8.